Gemfile

```diff
@@ -1,46 +1,46 @@
 source 'https://rubygems.org'
 
-gem 'rails'
-gem 'responders'
-gem 'mysql2'
-gem 'jbuilder'
-gem 'sdoc', group: :doc
-gem 'spring', group: :development
-gem 'coveralls', require: false
-gem 'faraday'
-
-# Use Squash for exception reporting
-gem 'squash_ruby', require: 'squash/ruby'
+gem 'rails', '~> 4.2' # specifying because we expect a major version upgrade to break things
 
 # Pinned to 1.3.3 until https://github.com/SquareSquash/rails/pull/15
 gem 'squash_rails', '1.3.3', require: 'squash/rails'
 
 # Application specific gems
+gem 'daemons' # TODO: where is this used?
 gem 'delayed_job_active_record'
-gem 'rest-client'
-gem 'daemons'
-gem 'whenever', require: false
 gem 'delayed_job_web'
+gem 'faraday'
 gem 'is_it_working-cbeer'
+gem 'jbuilder' # TODO: where is this used?
+gem 'rake' # needed for automation tasks
+gem 'responders' # TODO: where is this used?
+gem 'rest-client'
```

Comment:
Is `rest-client` still required in the Gemfile? I can't find a use of it in the application anymore.